spec/spec_helper.rb

```diff
@@ -18,11 +18,17 @@
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
   require 'simplecov'
-  SimpleCov.start 'rails'
   require 'capybara/rspec'
   require 'webmock'
   require 'vcr'
 
+  SimpleCov.start 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.